src/main/scala/com/databricks/labs/overwatch/validation/DeploymentValidation.scala

```diff
@@ -378,6 +378,13 @@ object DeploymentValidation extends SparkSessionWrapper {
           config.eh_scope_key,
           config.eh_name,
           config.output_path)
+      case "gcp" =>
+        validateAuditLog(
+          config.workspace_id,
+          config.auditlogprefix_source_aws,
+          config.primordial_date,
+          config.max_days
+        )
```

Comment:
added commit -- to avoid code redundancy merged logic into aws || gcp

Reply:
duplicate code removed